zone/command.cpp

```diff
@@ -9016,6 +9017,50 @@ void command_reloadtitles(Client *c, const Seperator *sep){
 
 }
 
+void command_allowexport(Client* c, const Seperator* sep) {
+	int arguments = sep->argnum;
+
+	if (!strcasecmp(sep->arg[1], "status")) {
+		uint8 flagval = c->GetCharExportFlag();
+		if (flagval == 0)
+			c->Message(CC_Default, "Export is currently off.");
+		else if (flagval == 1)
+			c->Message(CC_Default, "Export is currently set to include character info and worn items.");
+		else if (flagval == 2)
+			c->Message(CC_Default, "Export is currently set to include character info, worn items and inventory.");
+		else if (flagval == 3)
+			c->Message(CC_Default, "Export is currently set to include character info, worn items, inventory, and bank.");
+		return;
+	}
+	if (!strcasecmp(sep->arg[1], "off")) {
+		c->SetCharExportFlag(0);
+		return;
+	}
+	if (!strcasecmp(sep->arg[1], "worn")) {
+		c->SetCharExportFlag(1);
+		return;
+	}
+	if (!strcasecmp(sep->arg[1], "inventory")) {
+		c->SetCharExportFlag(2);
+		return;
+	}
+	if (!strcasecmp(sep->arg[1], "bank")) {
+		c->SetCharExportFlag(3);
+		return;
+	}
+
+	if (arguments) { c->Message(CC_Default, "Invalid argument"); }
```

Comment:
Nit: please keep this consistent with what I've been using ie; 
(preferred)

```
if (arguments) 
{ 
c->Message(CC_Default, "Invalid argument");
 }
```

or, un preferred
```
if (arguments) { 
c->Message(CC_Default, "Invalid argument");
 }
```
Makes it cleaner to read.